tests/unit/importer/conversion/test_template_manager.py

```diff
@@ -457,6 +452,47 @@ def test_do_import_with_default_values(self):
         self.assertEqual('extra field', result.get_content('extra_field'))
         self.assertEqual(errors, [])
 
+    def test_do_import_with_no_default_values(self):
+        # given:
+        row_template = self._create_row_template()
+
+        # and:
+        row = self._create_test_row()
+
+        # when:
+        result, errors = row_template.do_import(row)
+
+        # then:
+        self.assertEqual({'description': 'a thing used for writing', 'name': 'pen'}, result.content.as_dict())
+        self.assertEqual(errors, [])
+
+    def test_do_import_row_from_module_worksheet(self):
+        # given:
+        row_template = self._create_row_template()
+
+        # and:
+        row = self._create_test_row()
+
+        # when:
+        result, errors = row_template.do_import(row, is_module=True)
+
+        # then:
+        self.assertEqual(result.is_module, True)
+
+    def _create_row_template(self):
+        cell_conversions = [FakeConversion('name'), FakeConversion('description')]
+        row_template = RowTemplate('user', 'user', cell_conversions)
+        return row_template
+
+    def _create_test_row(self):
+        workbook = Workbook()
+        worksheet = workbook.create_sheet('list_of_things')
+        worksheet['A1'] = 'pen'
+        worksheet['B1'] = 'a thing used for writing'
```

Comment:
Is it not `A2` instead of `B1`?

Reply:
no , this is another property in the spreadsheet but same row